cli_tools/common/utils/compute/node_affinity_label_parser_test.go

```diff
@@ -42,7 +44,7 @@ func TestParseNodeAffinities(t *testing.T) {
 }
 
 func TestParseNodeAffinitiesSingleValue(t *testing.T) {
-	affinities, err := ParseNodeAffinityLabels([]string{"compute.googleapis.com/node-group-name,IN,zoran-playground-node-group"})
+	affinities, _, err := ParseNodeAffinityLabels([]string{"compute.googleapis.com/node-group-name,IN,zoran-playground-node-group"})
```

Comment:
this pattern indicates affinitiesbeta is not tested

Reply:
Base case is tested in TestParseNodeAffinities. I think it's low enough risk to assume other cases will behave the same for Beta